src/ducks/recurrence/DebugRecurrencePage.jsx

```diff
@@ -178,19 +184,22 @@ const updateBundles = (bundles, newTransactions, rules) => {
   const minDate = new Date(minBy(newTransactions, 'date').date)
   const dateSpan = (maxDate - minDate) / ONE_DAY
 
+  let updatedBundles
+
   if (dateSpan > 90 && newTransactions.length > 100) {
     const newBundles = findRecurringBundles(newTransactions, rules)
     const allBundles = [...bundles, ...newBundles]
-    const updatedBundles = groupBundles(allBundles, sameFirstLabel)
-    return updatedBundles
+    updatedBundles = groupBundles(allBundles, sameFirstLabel)
+    updatedBundles
   } else {
     const newBundles = newTransactions.map(t => ({ ops: [t] }))
     const allBundles = [...bundles, ...newBundles]
-    const updatedBundles = groupBundles(allBundles, sameFirstLabel)
-    return updatedBundles
+    updatedBundles = groupBundles(allBundles, sameFirstLabel)
+    updatedBundles.filter(bundles => bundles.ops.length > 1)
```

Comment:
Why do you have to filter here (in the else) if there is at least one ops and not in the if?

Reply:
I'll deal with this file in a later commit, and adding tests, at this point I'll have clearer ideas. Thanks.